api/db/changelog/db.changelog-115-addColumn-user-create-modified.xml

```diff
@@ -5,12 +5,17 @@
     xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
                     http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
   <changeSet author="nsaxena" id="db.changelog-115-addColumn-user-create-modified-with-null-constraint-and-default">
+    <preConditions onFail="MARK_RAN" > 
+      <not>         
+        <columnExists tableName="user" columnName="creation_time" />
+      </not>
+    </preConditions>
     <addColumn tableName="user">
-      <column name="creation_time" type="datetime" defaultValueDate="1970-01-01 00:00:00">
-        <constraints nullable="false" />
+      <column name="creation_time" type="datetime">
```

Comment:
We should just delete this changeset and start at 120 if it failed. I think that's more correct. 115 should never be in the history for two different databases with different semantics.

Reply:
what do you mean by 120?